src/components/membershipDAOList/membershipDAOList.tsx

```diff
@@ -0,0 +1,71 @@
+import React, {useState} from 'react';
+import {ButtonText, IconChevronDown} from '@aragon/ods-old';
+import {useTranslation} from 'react-i18next';
+import {EmptyMemberSection, MemberSection} from 'pages/daoMember';
+import {MemberDAOsType} from 'utils/types';
+import {ActionItemMembership} from 'components/membersList/actionItemMembership';
+
+export interface IMembershipDAOListProps {
+  daos: MemberDAOsType;
+}
+
+const initialDAOsPageSize = 3;
+const daosPageSize = 6;
+
+export const MembershipDAOList: React.FC<IMembershipDAOListProps> = ({
+  daos,
+}) => {
+  const {t} = useTranslation();
+
+  const [page, setPage] = useState(0);
+
+  const filteredDaos = daos?.slice(
+    0,
+    page === 0 ? initialDAOsPageSize : page * daosPageSize + initialDAOsPageSize
+  );
+  const hasMore = filteredDaos?.length < daos?.length;
+
+  if (daos?.length === 0) {
+    return (
+      <div className="flex w-full grow flex-col md:max-w-[400px]">
+        <EmptyMemberSection
+          title={t('members.profile.emptyState.Memberships')}
+          illustration="users"
+        />
+      </div>
```

Comment:
If we apply the layout that I specified below, this could simply be:
```suggestion
        <EmptyMemberSection
          title={t('members.profile.emptyState.Memberships')}
          illustration="users"
        />
```
Same for the code below, we could directly render the `<MemberSection />` component instead of the `<div />` wrapper